### PR TITLE
Update to 0.3.0 and enable phrases and substring

### PR DESCRIPTION
**Pull request by coltfred**
Updates to the 0.3.0 version and adds substring and phrases to the analyzer in the example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +69,8 @@ curl -s -G --data-urlencode "q=title:list" localhost:9200/try_cloaked_search/_se
 
 ### Sample Queries
 
+This is a couple examples of simple term queries:
```
Comment (clintfred) [inline README.md:72]:
```suggestion
Here are some examples of simple term queries:
```

File: README.md [hunk 2]
```diff
@@ -76,14 +79,29 @@ curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:Japan" localhos
 curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:cup" localhost:8675/try_cloaked_search/_search | jq
 ```
 
+Term queries can also be combined with ORs or ANDs like this:
+
```
Comment (clintfred) [inline README.md:83]: I wonder if it would be better to combine the search terms into a single subquery?

File: docker-compose.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ services:
       ES_JAVA_OPTS: "-Xmx1G"
       discovery.type: "single-node"
   cloaked-search:
-    image: "gcr.io/ironcore-images/cloaked-search:latest"
+    image: "gcr.io/ironcore-images/cloaked-search:0.3.0"
```
Comment (clintfred) [inline docker-compose.yml:11]: I think is rather leave this as latest so we don't have to update this with every release.

**Review by clintfred — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (giarc3) [inline README.md:72]:
```suggestion
These are a couple examples of simple term queries:
```

**Review by giarc3 — APPROVED** [review]

**Review by clintfred — APPROVED** [review]
With grammar fixed this can be merged. The tag and OR query comments are just suggestions.

File: README.md (hunk 2 above)
Comment (coltfred) [reply README.md:83]: In this case I'm not sure it makes much of a difference. I think it'd be nice to continue to expand this section to show more complex queries.

**Pull request merged**